tests/directives/escaped.js

```diff
@@ -0,0 +1,4 @@
+'\'';
+'\"';
+"\'";
```

Comment:
I'm trying to test that even unnecessarily escaped characters are preserved as-is, since that's what "exact code unit sequence" means (as I understand it). From the [original bug report](https://github.com/prettier/prettier/issues/1555#issue-227206837):

> For example, prettier formats
> ```js
> " \" ";
> ```
> [as](https://prettier.github.io/prettier/#%7B%22content%22%3A%22%5C%22%20%5C%5C%5C%22%20%5C%22%3B%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D)
> ```
> ' " ';
> ```
> but these are not the same directive, because they do not contain the same exact code unit sequence.